README.md

```diff
@@ -50,15 +40,31 @@ GRUB_TBOOT_POLICY_DATA=list.data
 
 2. Copy `20_linux_tboot` to `/etc/grub.d/20_linux_tboot`
 3. Run `grub2-mkconfig -o /etc/grub2.cfg`
-4. ~~Manually edit `/etc/grub2.cfg`:~~
-    1. ~~Add `--unrestricted` to the tboot entries~~
-    2. ~~Add `module /list.data` in the middle~~
-4. Make sure the SINIT is the last module loaded in the GRUB configuration
-5. Run `./create-lcp-tboot-policy_el7.sh $tpm_owner_password` to create and install policy
-6. Reboot
-7. Select tboot kernel module
 
-### Excerpt from working env:
+#### EL6 Grub Configuration
+
+1. Modify `/etc/grub.conf` as follows
+
+```
+title CentOS-tboot
+        root (hd0,0)
+        kernel /tboot.gz logging=serial,memory,vga vga_delay=1 min_ram=0x2000000
+        module /vmlinuz-2.6.32-696.el6.x86_64 ro root=/dev/mapper/vg_tpm04-lv_root rd_LVM_LV=vg_tpm04/lv_swap rd_NO_LUKS LANG=en_US.UTF-8 rd_NO_MD rd_LVM_LV=vg_tpm04/lv_root SYSFONT=latarcyrheb-sun16 crashkernel=auto KEYBOARDTYPE=pc KEYTABLE=us rd_NO_DM rhgb quiet intel_iommu=on
+        module /initramfs-2.6.32-696.el6.x86_64.img
+        #module /list.data
+        #module /2nd_gen_i5_i7_SINIT_51.BIN
+```
+
+## Enable TBOOT
+1. Reboot
+2. Boot into the tboot kernel. Tboot will fail, but this is expected.
+3. Own the TPM using the well-known SRK password:
+    `tpm_takeownership -z`
+4. Run `./create-lcp-tboot-policy_el<6,7>.sh $tpm_owner_password` to create and install policy
+  * In EL6, adjust `CMD_LINE` head -n based on grub.conf ordering.
```

Comment:
this can be made better in the script, and will need to be made better if we want puppet to do this